DrivewayDentDeletion/Operators/readme.md

```diff
@@ -50,8 +50,19 @@ These steps will need to be documented in the demo docs:
 ## Description of pipeline
 - Trigger: Whenever a commit is made to the forked repo it triggers the
   pipeline.
-- Clone git repo: Clones this git repo into a workspace PVC that is then used by the rest of the tasks in the pipeline.
+- Build tasks: Each of these tasks builds an images and pushes it to the cluster's local OpenShift Image Registry. The latest dockerfile and related files (bar files) are pulled from the forked git repo.
```

Comment:
Take a look at the readme here:
https://github.com/IBM/cp4i-deployment-samples/blob/ddd-apic/DrivewayDentDeletion/Operators/readme.md

Looks like there are some merge issues. Two copies of the section "Description of pipeline", two copies of the section "Pipeline with added support for dev/test environments". Also the APIC diagram needs to be updated with the "Clone git repo" task and match the formatting of the newer diagrams.